Backend/dev/test-tool/context-api/server.py

```
@@ -3998,6 +4062,171 @@ def do_DELETE(self):
+                            for ex in rpt.get("run", {}).get("executions", []):
+                                item = ex.get("item", {})
+                                resp = ex.get("response", {})
+                                url = resp.get("responseTime", 0)
+                                req = ex.get("request", {})
+                                req_url = req.get("url", {})
+                                raw = req_url.get("raw", "") if isinstance(req_url, dict) else str(req_url)
+                                # strip host, keep path
+                                try:
+                                    from urllib.parse import urlparse as _up
+                                    path = _up(raw).path or raw
+                                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)